src/applications/check-in/components/layout/Footer.jsx

```diff
@@ -29,49 +27,14 @@ const Footer = ({ router, isPreCheckIn }) => {
       >
         {t('need-help')}
       </h2>
-      {isPreCheckIn ? (
-        <div data-testid="pre-check-in-message">
-          <p>
-            <span className="vads-u-font-weight--bold">
-              {t(
-                'for-questions-about-your-appointment-or-if-you-have-a-health-related-concern',
-              )}
-            </span>{' '}
-            {t('call-your-va-provider')}
-            <br />
-            <ExternalLink href="/find-locations/" hrefLang="en">
-              {t('contact-your-va-provider')}
-            </ExternalLink>
-            .
-          </p>
-          <p>
-            <span className="vads-u-font-weight--bold">
-              {t('for-questions-or-help-with-pre-check-in')}
-            </span>{' '}
-            {t('please-call-our-myva411-main-information-line-at')}{' '}
-            <va-telephone contact={phoneNumbers.mainInfo} />{' '}
-            {t('and-select-0-were-here-24-7')}
-          </p>
-          <p>
-            {t('if-you-have-hearing-loss-call')}{' '}
-            <va-telephone contact="711" tty />.
-          </p>
+      {travelPages.includes(currentPage) ||
+      (currentPage && currentPage.includes('complete')) ? (
```

Comment:
this logic should exclude the complete page for pre-check-in.  You could bring back the isPreCheckIn prop or select the app from redux.